src/balancerd/src/lib.rs

```diff
@@ -157,12 +167,24 @@ impl BalancerService {
         let https = listen(&cfg.https_listen_addr).await?;
         let internal_http = listen(&cfg.internal_http_listen_addr).await?;
         let metrics = BalancerMetrics::new(&cfg);
+        let mut configs = ConfigSet::default();
+        configs = dyncfgs::all_dyncfgs(configs);
+        mz_dyncfg_ld::sync_ld_to_configset(
+            configs.clone(),
+            &BUILD_INFO,
+            |_| {},
```

Comment:
Can you configure the context with a new `cloud_provider_region` context that sets `cloud_provider` and `cloud_provider_region` keys, sourced from CLI flags? Would be nice to be able to target only balancers in a certain cloud provider region.